src/anomalib/data/utils/split.py

```diff
@@ -48,7 +47,7 @@ class ValSplitMode(str, Enum):
     SYNTHETIC = "synthetic"
 
 
-def concatenate_datasets(datasets: Sequence[AnomalibDataset]) -> AnomalibDataset:
+def concatenate_datasets(datasets: Sequence["data.AnomalibDataset"]) -> "data.AnomalibDataset":
```

Comment:
If I don't do this, then I get circular import errot. This is because the base dataset imports utils that is in split and Anomalib dataset is referred here.

Reply:
again, a future reference, but would it be possible to do like `anomalib.Dataset` if we have the relative imports properly